src/main/kotlin/graphics/scenery/backends/vulkan/VulkanTexture.kt

```diff
@@ -83,9 +82,11 @@ open class VulkanTexture(val device: VulkanDevice,
                 bufferImageCopy.bufferOffset(bufferOffset)
 
                 if(update != null) {
+                    logger.debug("Copying ${update.extents} to ${buffer.vulkanBuffer.toHexString()}")
```

Comment:
Needs to be converted to `{}` string representation.

Reply:
1b94c0e51492770cf11f44fb4e254bc57983b745 converts to string representation